src/library.cpp

```diff
@@ -501,6 +501,51 @@ void lammps_error(void *handle, int error_type, const char *error_text)
   }
 }
 
+/* ---------------------------------------------------------------------- */
+
+/** expand a single LAMMPS input line from a string.
+ *
+\verbatim embed:rst
+
+This function tells LAMMPS to expand the string in *cmd* like it would process
+an input line fed to :cpp:func:`lammps_command` **without** executing it.
+The entire string is considered as command and need not have a
```

Comment:
"as a command" ?

Reply:
copy-and-modify oversight. Should be improved now.